### DISCUSSION TITLE
Issues with Play Store on Stoney Ridge

### DISCUSSION THREAD

**Discussion by emceethemouth** [sebanc/brunch#1152] [Q&A]
Hi having artifacts on the Play Store using Stoney Ridge "grunt". Any ideas how to fix this?
![Screenshot 2021-07-16 9 30 03 AM](https://user-images.githubusercontent.com/7655090/125878124-d8344bec-aec4-43ef-bc10-e2689863aa59.png)
![Screenshot 2021-07-16 9 29 33 AM](https://user-images.githubusercontent.com/7655090/125878127-ee60a7d5-d440-4572-9840-0a76e79011f7.png)
 TIA!

**Comment by zainullahk** [comment]
same laptop same problem install cloudready instead
